xdsl/dialects/scf.py

```diff
@@ -14,9 +14,9 @@ class If(Operation):
     output: Annotated[VarOpResult, AnyAttr()]
     cond: Annotated[Operand, IntegerType.from_width(1)]
 
-    true_region = RegionDef()
+    true_region: Region
     # TODO this should be optional under certain conditions
```

Comment:
elaborate on conditions?

Reply:
I think this should be in another PR, since this one is unrelated to this.